script.js

```diff
@@ -22,16 +31,161 @@ ulCont.addEventListener('click', ((e) => {
+const popupsProjectMobile = document.querySelector('.popups-project-p-mobile');
+const tagsUp = document.querySelectorAll('.card-tagsLi');
+const tagsDown = document.querySelectorAll('.card-tagsLi1');
+
+const openModal = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[0].imagesrc2;
+  popupsProjectTitle.innerHTML = storeData[0].name;
+  popupsProjectDesktop.innerHTML = storeData[0].content1;
+  popupsProjectMobile.innerHTML = storeData[0].content2;
+  tagsUp.innerHTML = storeData[0].tags1;
+  tagsDown.innerHTML = storeData[0].tags2;
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn.addEventListener('click', openModal);
+
+const openModal2 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[1].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[1].name;
+  popupsProjectDesktop.innerHTML = storeData[1].content1;
+  popupsProjectMobile.innerHTML = storeData[1].content2;
+  popupsProjectTitle.style.fontSize = '20px';
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn2.addEventListener('click', openModal2);
+
+const openModal3 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[2].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[2].name;
+  popupsProjectDesktop.innerHTML = storeData[2].content1;
+  popupsProjectMobile.innerHTML = storeData[2].content2;
+  popupsProjectTitle.style.fontSize = '20px';
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn3.addEventListener('click', openModal3);
+
+const openModal4 = function () {
+  overlay.classList.remove('hidden');
+  modal.style.display = 'flex';
+  // eslint-disable-next-line no-use-before-define
+  popupsProjectImage.src = storeData[3].imagesrc;
+  popupsProjectTitle.innerHTML = storeData[3].name;
+  popupsProjectDesktop.innerHTML = storeData[3].content1;
+  popupsProjectMobile.innerHTML = storeData[3].content2;
+  document.documentElement.scrollTop = 0;
+};
+
+openModalBtn4.addEventListener('click', openModal4);
+
+const closeModal = function () {
+  modal.style.display = 'none';
+  overlay.classList.add('hidden');
+};
+
+closeModalBtn.addEventListener('click', closeModal);
+
+// Local Storage
+const nameEl = document.querySelector('.form-name');
+const nameVal = localStorage.getItem('nameEl');
+if (nameEl) {
+  nameEl.value = nameVal;
+}
+
+const mailEl = document.querySelector('.form-email');
+const emailVal = localStorage.getItem('mailEl');
+if (mailEl) {
+  mailEl.value = emailVal;
+}
+
+const commentsEl = document.querySelector('.form-section-textarea');
+const commentsVal = localStorage.getItem('commentsEl');
```

Comment:
- [ ] In order to be used after retrived from the local storage the saved string is turned back into an object using `JSON.parse` or similar methods. Kindly implement this. checkout the example below
![image](https://user-images.githubusercontent.com/81932478/210355163-e160cbb0-2402-4021-a618-c7ef8bac3607.png)